src/devtools/useAtomDevtools.ts

```diff
@@ -11,45 +11,45 @@ interface Config {
   autoPause?: boolean
 }
 
-interface Message {
+type Message = {
   type: string
   payload?: any
   state?: any
 }
 
-interface IConnectionResult {
+type ConnectionResult = {
```

Comment:
Sorry if this conflicts with your preference.
I'd use `interface` only when required: interface merging, module augmentation.

Reply:
This looks good, switching to `type` is probably better for the reasons you mentioned.